docs/topics/proc_exposing-application-metrics-using-prometheus-with-spring-boot.adoc

```diff
@@ -0,0 +1,175 @@
+
+[id='exposing-metrics-using-prometheus-with-spring-boot_{context}']
+= Exposing application metrics using Prometheus with {SpringBoot}
+
+In this example, you:
+
+* Configure your application to expose metrics.
+* Collect and view the data using Prometheus.
+
+Prometheus actively connects to a monitored application to collect data; the application does not actively send metrics to a server.
+
+.Prerequisites
+
+* Prometheus configured to collect metrics from the application:
++
+--
+. Download and extract the link:https://prometheus.io/download/[archive^] with the latest Prometheus release:
++
+[source,bash,options="nowrap"]
+----
+$ wget https://github.com/prometheus/prometheus/releases/download/v2.4.3/prometheus-2.4.3.linux-amd64.tar.gz
+$ tar -xvf  prometheus-2.4.3.linux-amd64.tar.gz
+----
+
+. Navigate to the directory with Prometheus:
++
+[source,bash,options="nowrap"]
+----
+$ cd  prometheus-2.4.3.linux-amd64
+----
+
+. Append the following snippet to the `prometheus.yml` file to specify the port where your application exposes the metrics endpoint.
++
+The `scrape_interval` setting is optional and sets Prometheus to collect metrics every 5 seconds.
++
+[source,yaml,options="nowrap"]
+----
+  - job_name: 'spring-actuator'
+    metrics_path: '/prometheus'
+    scrape_interval: 5s
+    static_configs:
+    - targets: ['localhost:8080']
+----
++
+If you have the `spring-boot-starter-actuator` dependency in your `pom.xml` file, {SpringBoot}-based applications expose metrics at the `/prometheus` endpoint by default.
+You can change the endpoint at which your application exposes the metrics by setting the `endpoints.prometheus.id` property to a different path, for example: `endpoints.prometheus.id=micrometheus`.
++
+The endpoint is secured by default.
+To ensure that metrics collection works in your application, you must:
+
+* include authentication in your Prometheus metrics collection endpoint, or
+* make the endpoint unsecured by setting a property, such as `management.security.enabled=false`.
+--
+
+* The Prometheus server started on `localhost`:
++
+--
+Start Prometheus and wait until the `Server is ready to receive web requests` message is displayed in the console.
+
+[source,bash,options="nowrap"]
+----
+$ ./prometheus
+----
+--
+
+.Procedure
+
+. Include the `spring-boot-starter-actuator`, `micrometer-registry-prometheus` and `micrometer-spring-legacy` dependencies in your `pom.xml`.
+The `micrometer-spring-legacy` dependency is required to provide Micrometer support for {SpringBoot} 1.5 applications.
```

Comment:
Can you also mention that this dependency is not required anymore for Spring Boot 2.x applications. @inoxx03